### DISCUSSION TITLE
Jaeger Monitoring Using Docker and Prometheus Metrics Exposure

### DISCUSSION THREAD

**Discussion by lxxxxxxk** [jaegertracing/jaeger#5353] [Q&A]
I am attempting to set up a Jaeger service with the following Docker command:
docker run --rm --name jaeger \
  -e COLLECTOR_ZIPKIN_HOST_PORT=:9411 \
  -p 6831:6831/udp \
  -p 6832:6832/udp \
  -p 5778:5778 \
  -p 16686:16686 \
  -p 4317:4317 \
  -p 4318:4318 \
  -p 14250:14250 \
  -p 14268:14268 \
  -p 14269:14269 \
  -p 9411:9411 \
  jaegertracing/all-in-one:1.56

After starting the service, I run the official demo with the following command:

docker run --rm -it --link jaeger --net monitor_backend  -p 8080-8083:8080-8083   -e OTEL_EXPORTER_OTLP_ENDPOINT="http://xxx.xxx.xxx.xx:4318"   jaegertracing/example-hotrod:1.56   all --otel-exporter=otlp

However, I've encountered issues with the monitoring functionality. I suspect there might be a configuration step related to Prometheus. Specifically, is there a need to expose a /metrics endpoint for monitoring purposes? If so, could you please advise where and how to configure this?

Thank you for your assistance.

**Comment by albertteoh** [comment]
Hey @lxxxxxxk https://github.com/jaegertracing/jaeger/tree/main/docker-compose/monitor contains instructions on how to bring up a working Monitoring stack. 

From there, you could inspect the [docker-compose.yml file](https://github.com/jaegertracing/jaeger/blob/main/docker-compose/monitor/docker-compose.yml) file to view the example configurations required to get monitoring working for Jaeger.
